site/plugins/site/extensions/components.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+use Kirby\Cachebuster\Cachebuster;
+use Kirby\Cms\App;
+use Kirby\Cms\FileModifications;
```

Comment:
There are several unused imports

Reply:
Several? For me only `FileModifications ` in this.

Reply:
I can see also in following classes:
- site/plugins/site/src/Algolia/Search.php
- site/plugins/site/src/CodeBlock/KirbyContent.php